datafusion/core/src/datasource/file_format/avro.rs

```diff
@@ -124,9 +129,10 @@ mod tests {
     #[tokio::test]
     async fn read_limit() -> Result<()> {
         let session_ctx = SessionContext::new();
-        let task_ctx = session_ctx.task_ctx();
+        let ctx = session_ctx.state();
```

Comment:
maybe `state` would be a better name than `ctx` as there is already `session_ctx` which is a Context 🤔